common/src/main/java/com/mx/path/core/common/accessor/UpstreamResponseError.java

```diff
@@ -0,0 +1,14 @@
+package com.mx.path.core.common.accessor;
+
+/**
+ * Use when the upstream response is an error that is not user-correctable
+ */
+public class UpstreamResponseError extends UpstreamResponseException {
```

Comment:
`UpstreamResponseUncorrectableUserException`?

Reply:
That makes it sound like it's a user exception.

Reply:
Added a little more to the description. Hopefully clearer.